### PR TITLE
Add XCFramework generation with Fastlane

### PR DESCRIPTION
**Pull request by apps4everyone**
you can easy generate and update the xcFramework with:
`bundle exec fastlane exportXCFramework`

used in PR:
https://github.com/FLEXTool/FLEX/pull/550

### CODE REVIEW

**Comment by apps4everyone** [comment]
@NSExceptional if you try in this branch:
`bundle install`
`bundle exec fastlane exportXCFramework`
a xcFramework is generated into /bundle

**Review by loveover98 — APPROVED** [review]

**Comment by NSExceptional** [comment]
Hey I'm sorry I've been busy this past week/end, I will get to this ASAP this week!

**Comment by apps4everyone** [comment]
any news here? maybe together with next patch update? thx

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
```
Comment (NSExceptional) [inline fastlane/Fastfile:14]:
```suggestion

```

File: fastlane/Appfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
+# apple_id("[[APPLE_ID]]") # Your Apple email address
```
Comment (NSExceptional) [inline fastlane/Appfile:2]: Should this be filled out? If not, do we need it in source control at all?

**Review by NSExceptional — CHANGES_REQUESTED** [review]

File: fastlane/Appfile (hunk 2 above)
Comment (apps4everyone) [reply fastlane/Appfile:2]: the values are not needed, we don't build and upload an IPA, we build only an XCFramework

File: fastlane/Fastfile (hunk 1 above)
Comment (apps4everyone) [reply fastlane/Fastfile:14]: this is generated by default by fastlane

Comment (apps4everyone) [reply fastlane/Fastfile:14]: but feel free to change...

**Comment by apps4everyone** [comment]
maybe for the xcFramework generation this can be used:
https://github.com/marketplace/actions/swift-create-xcframework

File: fastlane/Appfile (hunk 2 above)
Comment (apps4everyone) [reply fastlane/Appfile:2]: the value are marked as comment with # in the beginning, they are template files of fastlane. every fastlene user knows them

**Comment by NSExceptional** [comment]
Fastlane is dead

**Comment by apps4everyone** [comment]
don't think so. it will be only with new repo owner

**Comment by apps4everyone** [comment]
sad to see that PR's just getting closed without any discussion. Not very motivation for contributors 😞

**Comment by NSExceptional** [comment]
I'm not confident this will remain an industry standard tool for long. Let's give it time and revisit this if I'm wrong

Context: https://twitter.com/steipete/status/1627358489898000384?s=20

**Pull request closed without merging**